src/main/java/com/automatedtest/sample/pages/SettingPage.java

```diff
@@ -103,4 +107,38 @@ public String getSelectedRadioSpokenValue() {
         }
         return value;
     }
+
+    public void scrollToBottom() {
+        JavascriptExecutor js = ((JavascriptExecutor) this.driver);
+        js.executeScript("window.scrollTo(0, document.body.scrollHeight)");
+        this.wait.sleep(1);
+    }
+
+    public void scrollToTop() {
+        JavascriptExecutor js = ((JavascriptExecutor) this.driver);
+        js.executeScript("window.scrollTo(0, 0)");
+        this.wait.sleep(1);
+    }
+
+    public void takeSnapShot(int id) throws Exception {
+        TakesScreenshot scrShot = ((TakesScreenshot) this.driver);
+        File SrcFile = scrShot.getScreenshotAs(OutputType.FILE);
+
+        if (id == 1) {
+            FileUtils.copyFile(new File(SrcFile.getAbsolutePath()), new File(imagePath1));
+        }
+
+        if (id == 2) {
+            FileUtils.copyFile(new File(SrcFile.getAbsolutePath()), new File(imagePath2));
+        }
+    }
+
+    public boolean compareImage() {
```

Comment:
this should take file paths as parameter and then compare it
it should be in Common page as well